docs-chef-io/content/inspec/resources/selinux.md

```diff
@@ -0,0 +1,81 @@
++++
+title = "selinux resource"
+draft = false
+gh_repo = "inspec"
+platform = "linux"
+
+[menu]
+  [menu.inspec]
+    title = "selinux"
+    identifier = "inspec/resources/os/selinux.md selinux resource"
+    parent = "inspec/resources/os"
++++
+
+Use the `selinux` Chef InSpec audit resource to test the state/mode of SELinux policy.
+
+selinux resource extracts and exposes data reported by the command 'sestatus'
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+### Version
+
```

Comment:
Is there a relevant version number here?

Reply:
It won't be known until after release.